### ISSUE TITLE
Footer - Topic part of list items

### ISSUE THREAD

**Issue by davidstenger** [uswds/uswds#1659]
**Issue**:
The H4 Topic is part of the list items

**Solution**:
Remove the H4 for the list items

Example:
 ```html
<h4>Topic</h4>
<ul class="usa-unstyled-list usa-width-one-fourth usa-footer-primary-content">
  <li><a href="javascript:void(0);">Secondary link</a></li>
  <li><a href="javascript:void(0);">Secondary link</a></li>
  <li><a href="javascript:void(0);">Secondary link</a></li>
  <li><a href="javascript:void(0);">Secondary link</a></li>
</ul>
```

**Comment by maya** [comment]
This appears to have been fixed: the heading elements are now part of the list items.

However, it did seem to cause a regression #2344.

**Comment by maya** [comment]
Misread, reopening.

**Comment by maya** [comment]
Fixed in #2620.
